qless/workers/__init__.py

```diff
@@ -46,10 +47,10 @@ def title(cls, message=None):
     @classmethod
     def divide(cls, jobs, count):
         '''Divide up the provided jobs into count evenly-sized groups'''
-        jobs = list(zip(*itertools.izip_longest(*[iter(jobs)] * count)))
+        jobs = list(zip(*_compat.izip_longest(*[iter(jobs)] * count)))
         # If we had no jobs to resume, then we get an empty list
         jobs = jobs or [()] * count
-        for index in xrange(count):
+        for index in range(count):
```

Comment:
I understand, but most of our devs don't have a lot of Python 3 experience here. So, I'd rather not force us to maintain the shim if it needs to grow in the future. I think by using a standard compatibility layer will ease the burden for the Python 2 devs.